static/app/types/debugFiles.tsx

```diff
@@ -1,3 +1,5 @@
+import {ValidationErrorDetailed} from 'app/components/modals/debugFileCustomRepository/appStoreConnect/utils';
```

Comment:
this doesn't introduce a circular import but i'm not particularly confident about this import. maybe it's better to just duplicate the type here instead?

Reply:
should we move the `ValidationErrorDetailed` type and maybe add some `AppStoreConnect` prefix or rename it?

Reply:
do you have any ideas on where `ValidationErrorDetailed` could be moved to? i'm not too familiar with the front end's file structure so hints here would be very helpful.

Reply:
please do not worry... I worked on this in the PR https://github.com/getsentry/sentry/pull/29843